upgrade/1.4.4/README.md

```diff
@@ -0,0 +1,382 @@
+Helm charts:
+
+```bash
+cd "$CSM_DISTDIR"
+./lib/setup-nexus.sh ; echo "RC=$?"
+```
+
+On success, the output should end with the following:
+
+```text
++ Nexus setup complete
+setup-nexus.sh: OK
+RC=0
+```
+
+In the event of an error,
+consult [Troubleshoot Nexus](../../operations/package_repository_management/Troubleshoot_Nexus.md)
+to resolve potential problems and then try running `setup-nexus.sh` again. Note that subsequent runs of `setup-nexus.sh`
+may report `FAIL` when uploading duplicate assets. This is okay as long as `setup-nexus.sh` outputs `setup-nexus.sh: OK`
+and exits with status code `0`.
+
+### Update Argo CRDs
+
+(`ncn-m001#`) Run the following script in preparation for 1.4.4 patch upgrade:
+
+```bash
+for c in $(kubectl get crd | grep argo | cut -d' ' -f1)
+do
+   kubectl label --overwrite crd $c app.kubernetes.io/managed-by="Helm"
+   kubectl annotate --overwrite crd $c meta.helm.sh/release-name="cray-nls"
+   kubectl annotate --overwrite crd $c meta.helm.sh/release-namespace="argo"
+done
+```
+
+### Upgrade services
+
+(`ncn-m001#`) Run `upgrade.sh` to deploy upgraded CSM applications and services:
+
+```bash
+cd "$CSM_DISTDIR"
+./upgrade.sh
+```
+
+### Upload NCN images
+
+It is important to upload NCN images to IMS and to edit the `cray-product-catalog`. This is necessary when updating
+products with IUF. If this step is skipped, IUF will fail when updating or upgrading products in the future.
+
+(`ncn-m001#`) Execute script to upload CSM NCN images and update the `cray-product-catalog`.
+
+```bash
+/usr/share/doc/csm/upgrade/scripts/upgrade/upload-ncn-images.sh
+```
+
+### Upgrade Ceph and stop local Docker registries
+
+**Note:** This step is not necessary if it was already completed by the CSM `v1.3.5` patch, CSM `v1.4.2` patch, or CSM `V1.4.3` patch. If it was already run,
+the following steps can be re-executed to verify that Ceph daemons are using images in Nexus and the local Docker
+registries have been stopped.
+
+These steps will upgrade Ceph to `v16.2.13`. Then the Ceph monitoring daemons' images will be pushed to Nexus and the
+monitoring daemons will be redeployed so that they use these images in Nexus. Once this is complete, all Ceph daemons
+should be using images in Nexus and not images hosted in the local Docker registry on storage nodes. The third step
+stops the local Docker registry on all storage nodes.
+
+1. (`ncn-m001#`) Run Ceph upgrade to `v16.2.13`.
+
+   ```bash
+   /usr/share/doc/csm/upgrade/scripts/ceph/ceph-upgrade-tool.py --version "v16.2.13"
+   ```
+
+1. (`ncn-m001#`) Redeploy Ceph monitoring daemons so they are using images in Nexus.
+
+   ```bash
+   scp /usr/share/doc/csm/scripts/operations/ceph/redeploy_monitoring_stack_to_nexus.sh ncn-s001:/srv/cray/scripts/common/redeploy_monitoring_stack_to_nexus.sh
+   ssh ncn-s001 "/srv/cray/scripts/common/redeploy_monitoring_stack_to_nexus.sh"
```

Comment:
```suggestion
   ssh ncn-s001 /srv/cray/scripts/common/redeploy_monitoring_stack_to_nexus.sh
```

Reply:
This comment wasn't addressed in your earlier pass. Not sure if it was overlooked.

Reply:
I addressed the one on line 151, somehow I didn't see this one.